scripts/pre-build.js

```diff
@@ -5,8 +5,13 @@
  * right places, and transform it to make it ready to
  * be used by docusaurus.
  */
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').config();
+}
+
 const path = require('path');
 const { existsSync } = require('fs');
+const fs = require('fs-extra');
```

Comment:
I need to review when I'm deleting folders. I know that I liked it because it accepts globby patterns and then I could use `.gitignore` to clean all the files in the prebuild step but in this new PR it might not be needed.